client/src/models/Position.js

```diff
@@ -27,6 +27,21 @@ export default class Position extends Model {
     ADMINISTRATOR: "ADMINISTRATOR"
   }
 
+  static convertType(type) {
```

Comment:
This method can be moved to the bottom of the file where the other class methods are. The current structure is like the properties first and then the methods.